src/editsignatures/editsignatures.html

```diff
@@ -17,11 +17,38 @@
     <link href="editsignatures.css" rel="stylesheet" type="text/css" />
 </head>
 
-<body class="ms-font-1">
+<body class="ms-font-1" style="background-color:rgba(9, 210, 255, 0.192);">
     <main>
-        <div class="ms-font-1">Hello World!</div>
```

Comment:
Is this hello world message intentional? Is this a placeholder for something better in the future?

Reply:
It was not intentional.  I am not sure how that line got in there since it is not in my code ins Visual Studio.

Reply:
I believe that 'Hello World' was just a placeholder when I was showing Juan how the popout window opens a file.